lms/static/js/courseware/toggle_element_visibility.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function () {
```

Comment:
This should be done with RequireJS and [the `require_module` tag](https://github.com/edx/edx-platform/blob/master/common/djangoapps/pipeline_mako/templates/static_content.html#L61-L81).